pages/common/mpv.md

```diff
@@ -26,3 +26,7 @@
 - Play a file using a profile defined in the `mpv.conf` file:
 
 `mpv --profile {{profile_name}} {{file}}`
+
+- See your webcam/video0 output:
```

Comment:
We tend to try and avoid the 2nd person in tldr pages (i.e. `you`, `your`, etc). What about this instead?
```suggestion
- Display the output of webcam or other video input device:
```